projects/plugins/jetpack/_inc/client/recommendations/feature-utils.js

```diff
@@ -160,6 +173,21 @@ export const getStepContent = stepSlug => {
 				ctaText: __( 'Enable Site Accelerator', 'jetpack' ),
 				illustrationPath: '/recommendations/site-accelerator-illustration.svg',
 			};
+		case 'publicize':
+			return {
+				progressValue: '99',
+				question: __(
+					'Automatically share your posts to social media to grow your audience.',
+					'jetpack'
+				),
+				description: __(
+					'It’s easy to share your content to a wider audience by connecting your social media accounts to Jetpack. When you publish a post, it will automatically appear on all your favorite platforms. Best of all, it’s free. <ExternalLink>Learn more</ExternalLink>',
+					'jetpack'
+				),
+				descriptionLink: 'https://jetpack.com/2020/04/01/automate-social-media-sharing-wordpress/',
```

Comment:
I would recommend using `getRedirectUrl` here as well. PCYsg-pY7-p2 should help you create a new link.